lamindb/_context.py

```diff
@@ -179,9 +179,14 @@ def _track(
                         )
                     elif isinstance(e, NonInteractiveEditorError):
                         raise e
+                    elif isinstance(e, IndexError):
```

Comment:
Are there other possibilities for creating IndexError?

Can we wrap `nbproject.meta.live.title` and throw a `NotebookNoTitleError` or something like this?

We don't want notebooks without titles to be tracked.

I think this will only ever be raised during init phase, because if we raise an error it will guarantee that there is a `name` in the Transform table and  a title in the notebook.

If we don't raise an error, I fear we might corrupt notebook & database. [Corrrupt means: either no title in notebook or no name in Transform table].

Don't you think?

Reply:
Context: https://laminlabs.slack.com/archives/C04FPE8V01W/p1688380731268789?thread_ts=1688379577.407719&cid=C04FPE8V01W